djangocms_navigation/cms_menus.py

```diff
@@ -5,15 +5,21 @@
 from menus.base import Menu, Modifier, NavigationNode
 from menus.menu_pool import menu_pool
 
-from .models import MenuItem
+from .models import MenuContent, MenuItem
+from .utils import get_versionable_for_content
 
 
 class CMSMenu(Menu):
     def get_roots(self, request):
         # TODO: What if the MenuItem objects are related to an unpublished version?
```

Comment:
Now that this method is versioning aware, is this comment also resolved? Is this an issue at all? Should the tests test for this case more specifically - as in create multiple versions with multiple states? @chaosk Do you have any opinions?

Reply:
I think the method is version aware but todo is still relevant as let say published nav is unpublished then out method will return unpublish version of menu content as it was the latest version of it which still the case to handle? @chaosk